src/vs/workbench/contrib/tasks/common/problemMatcher.ts

```diff
@@ -790,6 +790,12 @@ export namespace Config {
 		*    the current working directory. This is the default.
 		*  - ["relative", "path value"]: the filename is always
 		*    treated relative to the given path value.
+		*  - "autodetect": the filename is treated relative to
```

Comment:
It is the opposite, the filename is treated absolute and then as relative if it doesn't exist on the absolute path.

Reply:
Are you sure? My understanding of this https://github.com/microsoft/vscode/blob/4d312af6e0a5670a717ff2c7ac0a9353f8934160/src/vs/workbench/contrib/tasks/common/problemMatcher.ts#L197 is that it's first trying relative, catching any error, returning if relative was found, then falling back to absolute otherwise.

Reply:
You're right, thanks for double checking and catching. I must have been half asleep when I looked before!